docs/source/api/tutorials/In Depth 2 - Drawing hyperedges.ipynb

```diff
@@ -57,20 +57,7 @@
    "outputs": [
```

Comment:
<p>done can be -&gt; can be done hehe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>